community/modules/compute/pbspro-execution/scripts/pbs_usecp.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: Configure mountpoints to use PBS local copy mechanism
+  become: true
+  hosts: localhost
+  vars:
+    pbs_home: /var/spool/pbs
+    mountpoints: ""
+  tasks:
+  - name: Allow PBS MoM to transfer files on shared filesystems using cp
+    ansible.builtin.lineinfile:
+      path: "{{ pbs_home }}/mom_priv/config"
+      line: $usecp *:{{ item }} {{ item }}
+    loop: "{{ mountpoints.split(',') }}"
+    notify:
+    - Restart PBS
+  - name: Ensure PBS is running
```

Comment:
nit: order of operations are not clear to me, but probably just because I am ansible noob. My understanding from https://docs.ansible.com/ansible/latest/user_guide/playbooks_handlers.html#controlling-when-handlers-run is that `Restart PBS` will not run until after `Ensure PBS is running`. Is that the desired behavior?

Reply:
By default, Ansible handlers run at the end of a play if-and-only-if they were notified. In this case, the notification will be generated any time the list of mount points changes (increase, decrease, rename).

I am not quite the Ansible expert and I'm going by memory, but I think you might need to pair "ensure it's running" with "restart it" because SystemD may complain about restarting a service that is not actually started. The typical SystemD implementation of restart is "stop then start" and stop will fail if not active.

Reply:
The apache example in your link demonstrates this pattern.

Reply:
"ACK"